src/test/resources/hudson/plugins/copyartifact/CopyArtifactTest/testPermissionWhenParameterized/config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Extracted from CopyArtifactTest.zip to avoid https://issues.jenkins-ci.org/browse/JENKINS-47075